### DISCUSSION TITLE
Pre-install some KDE applications to replace current GNOME applications

### DISCUSSION THREAD

**Discussion by everything411** [LingmoOS/Community#22] [Ideas]
for example `plasma-systemmonitor` to replace `gnome-system-monitor`
gnome applications cannot handle a scaling factor of 200% well under LingmoUI, whose GUI is too big with 200% scaling, while kde applications work well with 200% scaling. maybe we should prefer kde applications to gnome applications. also, kde applications are written in QT, potentially have better compatibility with LingmoUI, which is also QT-based

**Comment by vampywiz17** [comment]
I think Gnome Apps are temporary. The final goal to replace base apps to own "lingmo" app.
